lgc/state/PipelineState.cpp

```diff
@@ -296,7 +296,7 @@ void PipelineState::record(Module *module) {
   recordVertexInputDescriptions(module);
   recordColorExportState(module);
   recordGraphicsState(module);
-  if (m_palMetadata)
+  if (m_palMetadata && !isComputeLibrary())
```

Comment:
I think we want to setup metadata also for compute libraries?
While libraries do not have the usual metadata like register values, the driver needs to know how much stack/scratch the library needs and this is written into the metadata.